KeyValueStore/partition.ts

```diff
@@ -24,6 +24,12 @@ export function partition<V, M = undefined>(
 			return result as { value: V; retention?: isoly.TimeSpan; meta?: M } | undefined
 		},
 		list: async (options?: string | ListOptions): Promise<Continuable<ListItem<V, M>>> => {
+			if (options && typeof options != "string" && options.range) {
+				if (options.range[0] && options.range[0] != "")
+					options.range[0] = prefix + options.range[0]
+				if (options.range[1] && options.range[1] != "")
```

Comment:
`""` should be falsy so the `&& options.range[0] != ""` shouldn't be needed.
![image](https://user-images.githubusercontent.com/79835961/222681966-3754aba1-7342-43d4-838c-8b1103cb16a7.png)

Reply:
![image](https://user-images.githubusercontent.com/79835961/222681966-3754aba1-7342-43d4-838c-8b1103cb16a7.png)

Reply:
Ok, I have removed it.